src/components/Atoms/TextInputWithDropdown/TextInputWithDropdown.js

```diff
@@ -96,36 +121,37 @@ const TextInputWithDropdown = React.forwardRef(
       <Container
         className={`TextInputWithDropdown ${className}`.trim()}
         onKeyDown={navigateOptions}
+        ref={containerRef}
       >
         <Input
           {...inputProps}
           className="TextInputWithDropdown__input"
-          ref={ref}
+          ref={forwardedRef}
         />
-        {options.length > 0 && forceClosed === false && (
+        {options.length > 0 && !forceClosed && (
           <Options
             {...optionsProps}
             className="TextInputWithDropdown__options"
+            ref={dropdownRef}
+            hideBorder={hideBorder}
           />
         )}
       </Container>
     );
   }
 );
 
-const Options = ({
+const Options = React.forwardRef(({
   options,
   dropdownInstruction,
   onSelect,
   activeOption,
   resetActiveOption,
   ...rest
-}) => {
+}, ref) => {
   // Reset 'activeOption' when the list is unfocused.
   const onBlur = e => {
     const { target } = e;
-    // There's a delay before the new activeOption becomes the document.activeElement when
-    //  scrolling through the dropdown list via keyboard.
```

Comment:
Why remove this?

Reply:
AH dope. I was running through removing comments I'd left myself as I worked, and removed this too in the haste. Apologies

Reply:
always check those mad diffs